src/xlsx/xlsxzipreader.cpp

```diff
@@ -48,7 +48,11 @@ ZipReader::~ZipReader()
 
 void ZipReader::init()
 {
+#if QT_VERSION >= 0x050600
+    QVector<QZipReader::FileInfo> allFiles = m_reader->fileInfoList();
```

Comment:
Thank you, mabrand.
1) I don't think that fileInfoList always returns a QVector :
   /usr/local/**Qt-5.5.0**/include/QtGui/**5.5.0**/QtGui/private/qzipreader_p.h
   91:    **QList**\<FileInfo\> fileInfoList() const;
   /usr/local/**Qt-5.6.0**/include/QtGui/**5.6.0**/QtGui/private/qzipreader_p.h
   91:    **QVector**\<FileInfo\> fileInfoList() const;
2) It would indeed be a good idea to make this const.